cfg/challenges/grasping/grasping_game.yaml

```diff
@@ -20,6 +20,6 @@ llsfrb:
       phase: PRODUCTION
     # load data from latest game report with a given name
     # leave empty to always load from latest stored report
-    load-from-report: "GraspingChallenge"
+    load-from-report: "GraspingChallengeNew"
```

Comment:
Are we giving a new name for backwards compatibility (which should be broken anyhow)?

Reply:
i thought that this would avoid confusion also regarding existing databases.

Reply:
Fair point!